### ISSUE TITLE
Usage example in README.md is broken

### ISSUE THREAD

**Issue by eugene-dubovoy** [aio-libs/aiojobs#10]
In README.md we have following Usage example section:

```python
import asyncio
import aiojobs

async def coro(timeout):
    await asyncio.sleep(timeout)

async def main():
    scheduler = aiojobs.create_scheduler()
    for i in range(100):
        # spawn jobs
        await scheduler.spawn(coro(i/10))

    await asyncio.sleep(5.0)
    # not all scheduled jobs are finished at the moment

    # gracefully close spawned jobs
    await scheduler.close()

asyncio.get_event_loop().run_until_complete(main())
```

When I run it using python interpreter (i have Python 3.6.1 version), i have following error:

```
Traceback (most recent call last):
  File "test_background.py", line 50, in <module>
    asyncio.get_event_loop().run_until_complete(main())
  File "/Users/dubovoy/anaconda/lib/python3.6/asyncio/base_events.py", line 466, in run_until_complete
    return future.result()
  File "test_background.py", line 42, in main
    await scheduler.spawn(coro(i/10))
AttributeError: 'coroutine' object has no attribute 'spawn'
```

**Comment by eugene-dubovoy** [comment]
I've figured out that await is missing while creating scheduler.

PR #11 was created.

**Comment by asvetlov** [comment]
Fixed by #10
